watertap/unit_models/electrodialysis_0D.py

```diff
@@ -98,6 +116,75 @@ class Electrodialysis0DData(InitializationMixin, UnitModelBlockData):
     this must be False.""",
         ),
     )
+    CONFIG.declare(
+        "has_pressure_change",
+        ConfigValue(
+            default=False,
+            domain=In([True, False]),
+            description="Pressure change term construction flag",
+            doc="""Indicates whether terms for pressure change should be
+    constructed,
+    **default** - False.
+    **Valid values:** {
+    **True** - include pressure change terms,
+    **False** - exclude pressure change terms.}""",
+        ),
+    )
+    CONFIG.declare(
+        "pressure_drop_method",
+        ConfigValue(
+            default=PressureDropMethod.none,
+            domain=In(PressureDropMethod),
+            description="Method to calculate the frictional pressure drop in electrodialysis channels",
+            doc="""
+     **default** - ``PressureDropMethod.none``
+
+       .. csv-table::
+           :header: "Configuration Options", "Description"
+
+           "``PressureDropMethod.none``", "The frictional pressure drop is neglected." 
+           "``PressureDropMethod.experimental``", "The pressure drop is calculated by an experimental data as pressure drop per unit lenght."
+           "``PressureDropMethod.Darcy_Weisbach``", "The pressure drop is calculated by the Darcy-Weisbach equation."
+       """,
+        ),
+    )
+    CONFIG.declare(
+        "friction_factor_method",
+        ConfigValue(
+            default=FrictionFactorMethod.fixed,
+            domain=In(FrictionFactorMethod),
+            description="Method to calculate the Darcy's friction factor",
+            doc="""
+     **default** - ``FrictionFactorMethod.fixed``
+
+       .. csv-table::
+           :header: "Configuration Options", "Description"
+
+           "``FrictionFactorMethod.fixed``", "Friction factor is fixed by users" 
+           "``FrictionFactorMethod.Gurreri``", "Friction factor evaluated based on Gurreri's work"
```

Comment:
If the .rst documentation comes in a subsequent PR, these probably need cited

Reply:
Correct.  they will be cited in documentation and their citation has been given in the 1d doc.

Reply:
The reference and equations were added to the 0d .rst file.